angular-hotkeys/angular-hotkeys.d.ts

```diff
@@ -9,13 +9,16 @@ declare module ng.hotkeys {
 
     interface HotkeysProvider {
         template: string;
+        templateTitle:string;
         includeCheatSheet: boolean;
         cheatSheetHotkey: string;
         cheatSheetDescription: string;
 
-        add(combo: string, description: string, callback: (event: Event, hotkeys: ng.hotkeys.Hotkey) => void): void;
```

Comment:
Can you elaborate where it will break? The tests run fine and since the return type was void and the added arguments are optional I don't see where existing code will cease to compile.